packages/pyright-internal/src/analyzer/binder.ts

```diff
@@ -521,6 +522,7 @@ export class Binder extends ParseTreeWalker {
                 node.parameters.forEach((paramNode) => {
                     if (paramNode.name) {
                         const symbol = this._bindNameToScope(this._currentScope, paramNode.name);
+                        const docString = extractParameterDocumentation(ParseTreeUtils.getDocString(node?.suite?.statements ?? []) ?? "", paramNode.name.value) ?? undefined;
```

Comment:
This line of code is difficult to understand with all of the `?` and `??` operators. I recommend adding `if` statements to make it more understandable.